docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorting last shown persons : `sort`
```

Comment:
Did you forget to add `sort` to the `help` command?

Reply:
Oh yes, added it.